### PR TITLE
Pup AF1 Quest for Turbo Animator

### PR DESCRIPTION
**Pull request by pandorraffxi**
Creating the PUP AF1 Quest "The Wayward Automation" for Turbo Animator

Updated:
scripts\zones\Aht_Urhgan_Whitegate\npcs\Iruki-Waraki.lua  - Used to start/end quest
scripts\zones\Nashmau\npcs\Dnegan.lua - Updated CS for quest

Added:
scripts\zones\Caedarva_Mire\MobIDs.lua - Calls to get NM ID for quest in case of ID Shift
scripts\zones\Caedarva_Mire\mobs\Caedarva_toad.lua - created to update quest line on defeat
scripts\zones\Caedarva_Mire\npcs\qm9.lua - Was not present in DSP but is used to spawn NM/CS after NM death

### CODE REVIEW

**Comment by wrenffxi** [comment]
Please hold review on this PR.  Jumped the gun.  Will with Pandorra to get it up to spec.

**Comment by teschnei** [comment]
I'll close it for now, just drop a message when you're ready to have it reopened (I think we have to reopen it)

**Comment by pandorraffxi** [comment]
@teschnei  - Ready to reopen it when you're able

Thanks!

File: scripts/zones/Aht_Urhgan_Whitegate/TextIDs.lua [hunk 1]
```diff
@@ -16,11 +16,12 @@ ITEM_CANNOT_BE_OBTAINEDX = 221; -- You cannot obtain the ?Possible Special Code:
...
- ZASSHAL_DIALOG = 10993; -- 'ang about. Looks like the permit you got was the last one I 'ad, so it might take me a bit o' time to scrounge up some more. 'ere, don't gimme that look. I'll be restocked before you know it.
-MUSHAYRA_DIALOG = 4962; -- Sorry for all the trouble. Please ignore Hadahda the next time he asks you to do something.
- HADAHDA_DIALOG = 4913; -- Hey, think you could help me out?
- PAY_DIVINATION = 8765; -- You pay 1000 gil for the divination.
-
+ ZASSHAL_DIALOG         = 10993; -- 'ang about. Looks like the permit you got was the last one I 'ad, so it might take me a bit o' time to scrounge up some more. 'ere, don't gimme that look. I'll be restocked before you know it.
+MUSHAYRA_DIALOG         = 4962; -- Sorry for all the trouble. Please ignore Hadahda the next time he asks you to do something.
+ HADAHDA_DIALOG         = 4913; -- Hey, think you could help me out?
+ PAY_DIVINATION         = 8765; -- You pay 1000 gil for the divination.
+ YOU_CAN_BECOME_PUP     = 5831; -- You can now become a puppetmaster!
```
Comment (wrenffxi) [inline scripts/zones/Aht_Urhgan_Whitegate/TextIDs.lua]: Note to whomever eventually merges this.  This 5831 text id is pre- November 2017 version shift and won't be covered by DragonArmor's open PR

Comment (TeoTwawki) [reply scripts/zones/Aht_Urhgan_Whitegate/TextIDs.lua]: November update merged and I added this textID in so thsi PR won't need to. Rebase leaving this file out of the PR and yer good.

**Comment by TeoTwawki** [comment]
woops. sorry thought that was gonna work - I'll pull local and fix it so I can merge

**Comment by TeoTwawki** [comment]
I rebased for you since I screwed it up

**Pull request merged**